CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next release
 
+### Changed
+
+- 'Summary' dashboard: the 'Participation per project' and 'Participation per tag' work a little bit different. Now, if a project filter is active, the former will stay the same but highlight the selected project instead of showing the differences with other projects which were hard to interpret (analogous for 'Participation per tag').
 
```

Comment:
@luucvanderzee Seems like a very useful change, and makes sense to me. Just wanted to ask if Product had any thoughts on this? That graph does seem really strange/hard to interpret, but I just want to double check that we don't have any known customers that use it for something?

Reply:
I checked with Karel and he was cool with it!

Reply:
Also the new chart contains exactly the same information as the old one, except presented in an easier to understand matter (IMO at least) :)